src/modules/user-page/components/user-dashboard/UserDashboard.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Grid, Avatar, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+export default class UserDashboard extends React.Component {
+
+  render() {
+    if (this.props.userprofile) {
+      return (
+        <Grid className="user-dashboard" container alignItems={"center"} justify={"center"} direction={"row"}>
+          <Grid item>
+            <Avatar
+              alt={"user avatar"}
+              src={this.props.user.avatar}
+              className="user-avatar"
+            />
+          </Grid>
+          <Grid className="left-pdg" item>
+            <Grid className="nick-btn">
+              <div className="user-nickname">
+                {this.props.userprofile.nickname}
+              </div>
+              {this.props.userprofile._id === this.props.user._id ?
+                  <Button variant="outlined" id="dashboard-btn">
+                    <Link 
+                      className="link"
+                      to={`/edit-profile/${this.props.userprofile.nickname}`}
+                    >
+                      Edit profile
+                    </Link>
+                  </Button>
+                : ''}
+            </Grid>
+            <Grid className="all-infa" container>
+              <div><span>
+                {this.props.userprofile.posts ? this.props.userprofile.posts.length: "0"}
+              </span> posts</div>
+              <div className="followers"><span>
```

Comment:
add a class to spans like this

Reply:
<div className="quantity"><span className="number">

Reply:
<div className="quantity"><span className="number">